README.md

```diff
@@ -16,11 +16,11 @@ Install from npm:
 ## Usage
 
 ```javascript
-const ENS = require('../')
+const ENS = require('ethjs-ens')
 const HttpProvider = require('ethjs-provider-http')
 
 // For MetaMask or Mist compatibility:
-if (typeof window.web3 !== 'undefined') {
+if (typeof window === 'object' && window.web3) {
```

Comment:
This will throw an error of web3 is not defined, and so this is incorrect advice.

Reply:
It needs to check `window` first anyway because:
```console
$ node -e 'typeof window.web3'
[eval]:1
typeof window.web3
       ^

ReferenceError: window is not defined
    at [eval]:1:8
    at ContextifyScript.Script.runInThisContext (vm.js:23:33)
    at Object.runInThisContext (vm.js:95:38)
    at Object.<anonymous> ([eval]-wrapper:6:22)
    at Module._compile (module.js:571:32)
    at evalScript (bootstrap_node.js:391:27)
    at run (bootstrap_node.js:124:11)
    at run (bootstrap_node.js:427:7)
    at startup (bootstrap_node.js:123:9)
    at bootstrap_node.js:542:3
```

Also, my update is almost same with this, but I used just `window.web3` which is not enough maybe.

```diff
-if (typeof window.web3 !== 'undefined') {
+if (typeof window === 'object' && typeof window.web3 !== 'undefined') {
```

So I will replace that with the above :)